docs/.vuepress/components/PageDoc.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <Page :actionBarHidden="actionBarHidden">
+    <ActionBar title="My App" />
+    <StackLayout class="page-wrapper">
+      <StackLayout>
+        <Label text="My Content" />
+      </StackLayout>
+      <StackLayout orientation="horizontal">
+        <Label text="actionBarHidden:" />
+        <input type="checkbox" v-model="actionBarHidden" id="Page_actionBarHidden_input" />
+      </StackLayout>
+    </StackLayout>
+  </Page>
+</template>
+
+<script>
+import StackLayout from '../../../src/layouts/StackLayout';
+import Page from '../../../src/components/Page';
+import ActionBar from '../../../src/components/ActionBar';
+import Label from '../../../src/components/Label';
+
+export default {
+  name: 'PageDoc',
+  data() {
+    return {
+      actionBarHidden: false,
+      backgroundSpanUnderStatusBar: false,
+      statusBarStyle: 'light',
+    };
+  },
+  methods: {
```

Comment:
Navigate methods using only native. Please remove them? We should use own router plugin.

Reply:
They have been removed